test/test/jfr/JfrTests.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.jfr;
+
+import jdk.jfr.consumer.RecordedEvent;
+import jdk.jfr.consumer.RecordingFile;
+import one.profiler.test.Output;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JfrTests {
+
+    @Test(mainClass = RegularPeak.class)
+    public void regularPeak(TestProcess p) throws Exception {
+        Output out = p.profile("-e cpu -d 6 -f %f.jfr");
+        String jfrOutPath = p.getFile("%f").getAbsolutePath();
+        out = Output.convertJfrToCollapsed(jfrOutPath, "--to", "2500");
+        assert !out.contains("test/jfr/Cache\\.lambda\\$calculateTop\\$1");
+        out = Output.convertJfrToCollapsed(jfrOutPath,"--from", "2500", "--to", "5000");
+        assert out.samples("test/jfr/Cache\\.lambda\\$calculateTop\\$1") >= 1;
+        out = Output.convertJfrToCollapsed(jfrOutPath,"--from", "5000");
+        assert !out.contains("test/jfr/Cache\\.lambda\\$calculateTop\\$1");
+    }
+
+    /**
+     * Test to validate JDK APIs to parse Cpu profiling JFR output
+     *
+     * @param p The test process to profile with.
+     * @throws Exception Any exception thrown during profiling JFR output parsing.
+     */
+    @Test(mainClass = JfrCpuProfiling.class)
+    public void cpuProfiling(TestProcess p) throws Exception {
```

Comment:
The test is not about cpu profiling, but about parsing JFR recordings with the standard API. Name the method accordingly, e.g., `parseRecording`